erpnext/projects/doctype/task/task.py

```diff
@@ -199,21 +199,14 @@ def has_webform_permission(self):
 		if project_user:
 			return True
 
-	def set_is_overdue(self):
-		is_overdue = 1
+	def set_is_overdue(self, update=False):
```

Comment:
`update_modified` should be an argument. Also write a patch for tasks that had `status == "Overdue"`. Set `status = "Working"` and `is_overdue = 1`